src/crimson/common/errorator.h

```diff
@@ -596,14 +596,14 @@ struct errorator {
     }
 
     template <class FuncT>
-    auto finally(FuncT &&func) {
+    _future finally(FuncT &&func) {
```

Comment:
this commit was revised based on the commit of "crimson/common/errorator: fix finally" in #38290

see also https://github.com/ceph/ceph/pull/38290/commits/7d2817ae3ba6be8693b5de11079d361da9c49c80#r530939728